<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDevice</class>
 <widget class="QWidget" name="AddDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 255);
font: 75 72pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="title">
    <string>添加选项:</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>开关类</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>窗帘类</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>调光类</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>红外类</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>315类</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>其他类</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>631</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 72pt &quot;Ubuntu&quot;;
color: rgb(85, 170, 0);
gridline-color: rgb(255, 170, 127);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>触发</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 127);
font: 75 72pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>雨蓬类</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 75 72pt &quot;Ubuntu&quot;;
color: rgb(255, 170, 255);</string>
    </property>
    <property name="text">
     <string>名称:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 72pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 127);
border-color: rgb(85, 170, 127);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 170, 255);
font: 75 72pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>位置:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 72pt &quot;Ubuntu&quot;;
border-color: rgb(85, 170, 127);
color: rgb(0, 0, 127);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>380</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);
font: 75 72pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);
font: 75 72pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 72pt &quot;Ubuntu&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>查看</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>380</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 75 72pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>门锁类</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>指纹类</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <zorder>radioButton</zorder>
   <zorder>radioButton_2</zorder>
   <zorder>radioButton_3</zorder>
   <zorder>radioButton_4</zorder>
   <zorder>radioButton_5</zorder>
   <zorder>radioButton_6</zorder>
   <zorder>tableWidget</zorder>
   <zorder>pushButton</zorder>
   <zorder>radioButton_7</zorder>
   <zorder>label</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_2</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>radioButton_8</zorder>
   <zorder>radioButton_9</zorder>
   <zorder>comboBox_3</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 0, 0);
font: 75 72pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>.....</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
